pkg/operator/controller/networkpolicyregistrar/add.go

```diff
@@ -37,34 +35,9 @@ func (r *Reconciler) AddToManager(mgr manager.Manager) error {
 	return builder.
 		ControllerManagedBy(mgr).
 		Named(ControllerName).
-		For(&operatorv1alpha1.Garden{}, builder.WithPredicates(NetworkingPredicate())).
+		For(&operatorv1alpha1.Garden{}, builder.WithPredicates(predicateutils.ForEventTypes(predicateutils.Create))).
 		WithOptions(controller.Options{
 			MaxConcurrentReconciles: 1,
 		}).
 		Complete(r)
 }
-
-// NetworkingPredicate returns true for Create and Update events if the
-// garden.Spec.RuntimeCluster.Networking.{Pods,Services} fields are not empty
-// TODO(shafeeqes): Remove this predicate in v1.75
```

Comment:
Please do not get me wrong. I am all for removing unnecessary code, but how come we wanted to remove it in v1.75 and now we remove it in v1.73? Are we that fast ;-) ?

Reply:
Ideally, The garden CRD (the served API) should be upgraded before the controllers. So all the versions after v1.71 should have the networking field, and they would be added to the reconciler without any issue as mentioned in the description. When I introduced the TODO, I didn't give it much thought but just calculated 3-4 versions, so that's why it's v1.75.